tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -451,6 +451,28 @@ def from_tensor_slices(tensors):
     # Dictionary structure is also preserved.
     Dataset.from_tensor_slices({"a": [1, 2], "b": [3, 4], "c": [5, 6]})
     # ==> [ {"a": 1, "b": 3, "c": 5}, {"a": 2, "b": 4, "c:" 6} ]
+
+
+    # Example 1: Two tensors can be combined into one Dataset object.
+    features = tf.constant([[1, 3], [2, 3], [2, 1], [1, 2], [3, 3], [3, 2]]) # ==> 6x2 tensor
```

Comment:
this exceeds the 80 character line limit, can we shorten the example to use a 3x2 tensor?

Reply:
> Thanks for contributing back these examples @splovyt! I left a couple comments

Sure thing, thanks for the review!
I have applied your suggestions; let me know if there's anything else you would like to see changed.